src/main/webapp/help-anonymous-access.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    When this box is checked, the plugin will force users to log in via Kerberos for all URLs they visit. This has the side effect that "log out" is essentially ineffective.<br />
```

Comment:
Shouldn't it be the other way around!?
At least I think by default it should behave as it used to, (default false) so that upgrading users don't get surprised about new behaviour.